packages/big-design/src/components/Alert/spec.tsx

```diff
@@ -1,97 +1,111 @@
 });
 
-test('render error Alert', () => {
-  const { container } = render(<Alert messages={[{ text: 'Error' }]} type="error" />);
+test('render error Alert', async () => {
+  render(<Alert messages={[{ text: 'Error' }]} type="error" />);
+
+  const alert = await screen.findByRole('alert');
 
-  expect(container.firstChild).toMatchSnapshot();
-  expect(container.firstChild).toHaveStyle(`border-left: ${theme.spacing.xxSmall} solid ${theme.colors.danger}`);
+  expect(alert).toMatchSnapshot();
+  expect(alert).toHaveStyle(`border-left: ${theme.spacing.xxSmall} solid ${theme.colors.danger}`);
 });
 
-test('render warning Alert', () => {
-  const { container } = render(<Alert messages={[{ text: 'Warning' }]} type="warning" />);
+test('render warning Alert', async () => {
+  render(<Alert messages={[{ text: 'Warning' }]} type="warning" />);
+
+  const alert = await screen.findByRole('alert');
 
-  expect(container.firstChild).toMatchSnapshot();
-  expect(container.firstChild).toHaveStyle(`border-left: ${theme.spacing.xxSmall} solid ${theme.colors.warning50}`);
+  expect(alert).toMatchSnapshot();
+  expect(alert).toHaveStyle(`border-left: ${theme.spacing.xxSmall} solid ${theme.colors.warning50}`);
 });
 
-test('render info Alert', () => {
-  const { container } = render(<Alert messages={[{ text: 'Info' }]} type="info" />);
+test('render info Alert', async () => {
+  render(<Alert messages={[{ text: 'Info' }]} type="info" />);
 
-  expect(container.firstChild).toMatchSnapshot();
-  expect(container.firstChild).toHaveStyle(`border-left: ${theme.spacing.xxSmall} solid ${theme.colors.primary60}`);
-});
+  const alert = await screen.findByRole('alert');
 
-test('renders with link', () => {
-  const { queryByRole, container } = render(
-    <Alert messages={[{ text: 'Success', link: { text: 'Link', href: '#' } }]} />,
-  );
+  expect(alert).toMatchSnapshot();
+  expect(alert).toHaveStyle(`border-left: ${theme.spacing.xxSmall} solid ${theme.colors.primary60}`);
+});
 
-  expect(container.firstChild).toMatchSnapshot();
+test('renders with link', async () => {
+  render(<Alert messages={[{ text: 'Success', link: { text: 'Link', href: '#' } }]} />);
 
-  const link = queryByRole('link') as HTMLAnchorElement;
+  const alert = await screen.findByRole('alert');
+  const link = await screen.findByRole<HTMLAnchorElement>('link');
 
+  expect(alert).toMatchSnapshot();
   expect(link).toBeInTheDocument();
   expect(link.href).toBe('http://localhost/#');
 });
 
-test('renders with external link', () => {
-  const { queryByRole, container } = render(
+test('renders with external link', async () => {
+  render(
     <Alert messages={[{ text: 'Success', link: { text: 'Link', href: '#', external: true, target: '_blank' } }]} />,
   );
 
-  expect(container.firstChild).toMatchSnapshot();
-
-  const link = queryByRole('link') as HTMLAnchorElement;
+  const alert = await screen.findByRole('alert');
+  const link = screen.getByRole('link') as HTMLAnchorElement;
```

Comment:
Let's remove this cast too.
```suggestion
  const link = await screen.fihdByRole<HTMLAnchorElement>('link');
```

Reply:
Totally agree with removing cast here. But It looks like we don’t have async code here so what’re your thoughts on using a `getBy*` instead?
 
`const link = screen.getByRole<HTMLAnchorElement>('link')`

Reply:
I guess that's alright 👍